docs/debugger/intellij.rst

```diff
@@ -0,0 +1,105 @@
+Debug Open edX with IntelliJ IDEA and Add a Python Remote SDK
+==============================================================
+
+To debug Open edX with IntelliJ IDEA and add a Python remote SDK, follow these steps:
```

Comment:
We can specify in what cases we would need a Python Remote SDK. I expect it is needed for IntelliJ as a general but not for PyCharm?